p2p/peer.py

```diff
@@ -359,6 +359,11 @@ def process_p2p_handshake(self, cmd: protocol.Command, msg: protocol._DecodedMsg
             self.disconnect(DisconnectReason.other)
             raise HandshakeFailure("Expected a Hello msg, got {}, disconnecting".format(cmd))
         remote_capabilities = msg['capabilities']
+
+        # if current_peers
+        #     self.disconnect(DisconnectReason.too_many_peers)
```

Comment:
For right now, I'm ok with making this a TODO and opening up an issue to enforce the max at a later date.